### PR TITLE
8345063: Centralize version number of the Gradle build file

### PR DESCRIPTION
**Pull request by nlisker**
Centralizes version numbers in a single place. This is not the best way of doing it in Gradle, but it's an improvement that will simplify other changes. I used the `build.properties` file because other versions are already written there.

There are other version numbers in the build file, but they are not attached directly to a dependency. Mostly libav ones. I decided not to touch them.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8345063](https://bugs.openjdk.org/browse/JDK-8345063): Centralize version number of the Gradle build file (**Enhancement** - P4)

### Reviewers
 * [Kevin Rushforth](https://openjdk.org/census#kcr) (@kevinrushforth - **Reviewer**)
 * [Marius Hanl](https://openjdk.org/census#mhanl) (@Maran23 - Committer)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx.git pull/1649/head:pull/1649` \
`$ git checkout pull/1649`

Update a local copy of the PR: \
`$ git checkout pull/1649` \
`$ git pull https://git.openjdk.org/jfx.git pull/1649/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1649`

View PR using the GUI difftool: \
`$ git pr show -t 1649`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx/pull/1649.diff">https://git.openjdk.org/jfx/pull/1649.diff</a>

</details>
<details><summary>Using Webrev</summary>

[Link to Webrev Comment](https://git.openjdk.org/jfx/pull/1649#issuecomment-2501265690)
</details>

### CODE REVIEW

File: build.properties [hunk 1]
```diff
@@ -106,3 +106,18 @@ jfx.build.macosx.xcode.version=Xcode14.3.1+1.0
...
+junit.version=4.13.2
+junit.jupiter.version=5.8.1
+junit.platform.version=1.8.1
+hamcrest.version=1.3
+opentest4j.version=1.2.0
+apiguardian.version=1.1.2
+
+icu.version=74.2
+antlr4.version=4.7.2
+lucene.version=7.7.3
```
Comment (kevinrushforth) [inline build.properties:123]: Minor: Add a (single) blank line at the end of this file?

**Review by kevinrushforth — COMMENTED** [review]
The changes look good. I'll do a quick CI test build before approving.

**Review by kevinrushforth — APPROVED** [review]
Looks good.

**Review by Maran23 — APPROVED** [review]
I agree that this is a good idea and something I do as well in Maven and Gradle projects. Putting them in the `build.properties` is fine.
Changes look good to me.

**Pull request closed without merging**